apps/wallet-mobile/src/features/Staking/Governance/useCases/ChangeVote/ChangeVoteScreen.tsx

```diff
@@ -171,20 +173,24 @@ export const ChangeVoteScreen = () => {
   )
 }
 
-const styles = StyleSheet.create({
-  root: {
-    paddingHorizontal: 18,
-    flex: 1,
-    justifyContent: 'space-between',
-  },
-  description: {
-    fontFamily: 'Rubik-Regular',
-    fontSize: 16,
-    lineHeight: 24,
-    color: '#242838',
-  },
-  actions: {
-    flex: 1,
-    gap: 16,
-  },
-})
+const useStyles = () => {
+  const {color, atoms} = useTheme()
+  const styles = StyleSheet.create({
+    root: {
+      ...atoms.flex_1,
+      ...atoms.justify_between,
+      paddingHorizontal: 18,
```

Comment:
@michaeljscript for everything that is not following the standards, from now on, approximate to the closest one
```suggestion
      ...atoms.px_lg,
```